app/helpers/template_helpers.py

```diff
@@ -117,37 +207,34 @@ def get_census_base_url(schema_theme: str, language_code: str) -> str:
     return CENSUS_BASE_URL
 
 
-def get_contact_us_url(language_code: str, base_url: str):
-    if language_code == "cy":
-        return f"{base_url}cysylltu-a-ni/"
-
-    return f"{base_url}contact-us/"
-
-
-def get_footer_urls(language_code: str, base_url: str, schema_theme: str):
+def get_static_content_urls(language_code: str, base_url: str, schema_theme: str):
```

Comment:
This method should have the same cache decorator (`@lru_cache(maxsize=None)`) as the `get_page_header_context` call. This ensures that the urls are only generated once for any given combination of arguments.